src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+      key: B
+      order: 1
+    - content: One email address to another.
+      key: C
+      order: 2
+    - content: One crypto wallet to another without a third party observation.
+      key: D
+      order: 3
+- uuid: 93e24b6c-9a74-4bae-8c93-c439620fe67e
+  type: SingleChoice
+  content: What is the Total value locked in DeFi as of November 2021?
+  hint: NoHint
+  explanation: TVL or Total value locked refers to total stored value in the blockchain. Currently, DeFi accomodates more than $40 Billion of worth in it.
+  answerKeys:
+    - D
+  subTopics:
+    - "introduction-to-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: $20 Billion
+      key: A
+      order: 0
+    - content: $60 Billion
+      key: B
+      order: 1
+    - content: $30 Billion
+      key: C
+      order: 2
+    - content: $40 Billion
+      key: D
+      order: 3
+- uuid: cd6b29cb-ad5a-4f1a-8d44-a780ae9c8bd9
+  type: SingleChoice
+  content: What do you understand by interoperability of DeFi?	
+  hint: Features of Decentralised Finance
+  explanation: Interoperability of DeFi refers to interconnection between various dapps for fast and easy transfer of assets.
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: DeFi is accessible from anywhere.
+      key: A
+      order: 0
+    - content: DeFi is free to use for everyone.
+      key: B
+      order: 1
+    - content: DeFi applications are interconnected.
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: b7ffffa0-4478-40fb-8fbf-32463792876a
+  type: SingleChoice
+  content: How does DeFi solve the volatility of cryptocurrency?
+  hint: Uses of DeFi
+  explanation: Stablecoins do not fluctuate because they are backed by a highly stable asset such as gold or USD.
+  answerKeys:
+    - B
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: Using smart contracts
+      key: A
+      order: 0
+    - content: Using Stablecoins
+      key: B
+      order: 1
+    - content: Using Flash Loans
+      key: C
+      order: 2
+    - content: None of these
+      key: D
+      order: 3
+- uuid: a63ed714-02dc-4f2c-956b-8dac94e583f9
+  type: SingleChoice
+  content: Centralised Finance provides a transparent system to its customers.
+  hint: Disadvantages of Centralised system
+  explanation: Centralised finance hides its functioning and procedures from its users which creates confusion and many times hidden charges for them also.
+  answerKeys:
+    - B
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 2c93b4b7-d351-4a32-aaaa-5170524b04dc
+  type: MultipleChoice
+  content: How does DeFi help young entrepreneurs?
```

Comment:
From the choices below nothing specifically applies to `young entrepreneurs?`

Reply:
Young Entrepreneurs -> Small scale businesses